bugbug/repository.py

```diff
@@ -211,6 +214,23 @@ def _transform(commit):
             type_ = ext
         obj["types"].add(type_)
 
+        try:
```

Comment:
Maybe it'd be faster to keep # of added/removed lines for each revision and after getting all the log, computing the size of the file for each revision.

Reply:
I've thought about that, but then we'd need to analyze all the revisions in sequence and we couldn't use multiple processes to analyze them.

Reply:
I'd guess that it's probably faster to have two runs even if the second is single threaded.
Maybe we could write an extension to just get the '\n' count without retrieving all the file.

Reply:
Yeah, but in the end we'll probably need to hg.cat the file anyway for #371 and other parts of #372, so it might not be worth optimizing this now if we have to do this anyway later.